docs/self-managed/deployment/helm/upgrade/helm-870-880.md

```diff
@@ -96,28 +97,186 @@ Because of to Bitnami’s support policy changes, Camunda now uses enterprise im
+- Only required when using the specific management and modeling components
 
-#### LDAP
+#### Identity mangement migration guidance
 
-No migration path is available. Please contact Camunda Consulting for support.
+The changes affect different deployment scenarios:
+
+- **Basic Authentication**: Previously applied only to Operate/Tasklist, now applies to the entire Orchestration Cluster
+- **OIDC/OAuth setups**: Orchestration Cluster now authenticates directly with your identity provider
+- **User migration**: Existing users and authorizations need to be migrated to the orchestration cluster identity
+
+:::caution Keep your existing authentication method
+**You must keep the same authentication method you used in 8.7.** Changing authentication methods during the upgrade is not supported. Your 8.8 configuration must use one of the following methods matching your 8.7 setup:
+
+- **Basic Authentication**: Built-in user management
+- **External Identity Provider (OIDC)**: Integration with enterprise identity systems (Microsoft Entra ID, Okta, etc.)
+- **Internal Keycloak**: Managed Keycloak instance
+  :::
+
+#### Basic authentication
+
+Basic authentication is enabled by default and provides built-in user management for the Orchestration Cluster.
+
+**Key changes in 8.8:**
+
+- Basic auth now applies to the **entire Orchestration Cluster** (not just Operate/Tasklist as in 8.7)
+- All API requests to the cluster require authentication unless explicitly disabled (including Zeebe)
+- Users must be manually configured in the new Orchestration Cluster Identity
+  - Ensure usernames remain identical to preserve task assignments
+  - Update client applications to authenticate with the unified cluster
+
+**Configuration steps:**
+
+1. **Enable/disable authentication**: To disable authentication entirely (not recommended for production):
+
+   ```yaml
+   orchestration:
+     security:
+       authentication:
+         method: none
+   ```
+
+2. **Configure users**: Add users to the Orchestration Cluster configuration:
+
+   ```yaml
+   orchestration:
+     security:
+       initialization:
+         users:
+           - username: demo
+             password: demo
+             name: Demo User
+             email: demo@demo.com
+   ```
+
+3. **Assign roles**: Configure default roles for users:
+   ```yaml
+   orchestration:
+     security:
+       initialization:
+         defaultRoles:
+           admin:
+             users:
+               - demo
+   ```
+
+For detailed basic authentication configuration, see [Basic authentication setup](../../configure/authentication-and-authorization/#basic-authentication).
+
+#### External identity provider (OIDC)
+
+Connect the Orchestration Cluster to your OIDC-compatible identity provider (Microsoft Entra ID, Okta, etc.).
+
+**Two authentication scopes:**
+
+1. **Orchestration Cluster authentication** (new in 8.8):
+   - Zeebe, Operate, and Tasklist authenticate directly with your OIDC provider
```

Comment:
It is always authenticated via the OIDC provider. What is the purpose of this point?